### PR TITLE
Add Informative geolocation text for virtual card

### PR DESCRIPTION
**Pull request by mauricecarrier7**
**What's this do?**
Adds copy informing user about why Palace requests geo location and how to permit access.

**Why are we doing this? (w/ Notion link if applicable)**
https://www.notion.so/lyrasis/iOS-Add-prompt-informing-user-about-geolocation-request-f1e6488bb92a4fccb27e2e18ec3e9c62?pvs=4

**How should this be tested? / Do these changes have associated tests?**
Got to an account that supports virtual library card creation. Copy should be visible, button should request permissions.

**Dependencies for merging? Releasing to production?**
N/A

**Does this include changes that require a new Palace build for QA?**
Yes

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@mauricecarrier7

### CODE REVIEW

File: Palace/Utilities/System/LocationManager.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  private let locationManager = CLLocationManager()
+
+  private override init() {
+    super.init()
+    locationManager.delegate = self
+  }
+  
+  var locationAccessAuthorized: Bool {
+    let status = locationManager.authorizationStatus
+    return status == .authorizedAlways || status == .authorizedWhenInUse
```
Comment (vladimirfedorov) [inline Palace/Utilities/System/LocationManager.swift:26]: When `authorizationStatus` is `.notDetermined`, the app shows "Open Settings" message when the app should ask the permission instead: on the first run (in this case Location access doesn't exist in the settings, so the user can't enable it) and when the user changes the value in the settings to "Ask next time or when I share".

**Review by vladimirfedorov — CHANGES_REQUESTED** [review]

File: Palace/Views/RegistrationCell.swift [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    regTitle.text = title ?? regTitle.text
+    regBody.text = body ?? regBody.text
+    regButton.setTitle(buttonTitle ?? regButton.titleLabel?.text ?? "", for: .normal)
+    regButton.addAction(UIAction(handler: { _ in  buttonAction() }), for: .touchUpInside)
+  }
+
+  @objc func configureForDeniedLocationAccess() {
+    let attributedString = NSMutableAttributedString(string: DisplayStrings.deniedLocationAccessMessage)
+    let boldRange = (DisplayStrings.deniedLocationAccessMessage as NSString).range(of: DisplayStrings.deniedLocationAccessMessageBoldText)
+    let boldFont = UIFont.boldPalaceFont(ofSize: UIFont.systemFontSize + 2.0)
```
[the remark this answers is not in the document]
Comment (mauricecarrier7) [reply Palace/Views/RegistrationCell.swift]: I bumped it two more points

File: Palace/Utilities/System/LocationManager.swift (hunk 1 above)
Comment (mauricecarrier7) [reply Palace/Utilities/System/LocationManager.swift:26]: nice catch, updated

File: Palace/Views/RegistrationCell.swift (hunk 2 above)
[the remark this answers is not in the document]
Comment (mauricecarrier7) [reply Palace/Views/RegistrationCell.swift]: I misunderstood your iniital statement, I'll remove the size increase

**Review by vladimirfedorov — APPROVED** [review]
Looks good to me

**Pull request merged**